egov-searcher/src/main/java/org/egov/custom/mapper/billing/impl/BillRowMapper.java

```diff
@@ -0,0 +1,147 @@
+package org.egov.custom.mapper.billing.impl;
+
+import java.math.BigDecimal;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.egov.custom.mapper.billing.impl.Bill.StatusEnum;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.jdbc.core.ResultSetExtractor;
+import org.springframework.stereotype.Component;
+import org.springframework.util.CollectionUtils;
+import org.springframework.web.client.RestTemplate;
+
+@Component
+public class BillRowMapper implements ResultSetExtractor<List<Bill>>{
+	
+	@Autowired
+	private RestTemplate rest;
+	
+	@Value("${egov.user.contextpath}")
+	private String userContext;
+	
+	@Value("${egov.user.searchpath}")
+	private String userSearchPath;
+
+	@Override
+	@SuppressWarnings("unchecked")
+	public List<Bill> extractData(ResultSet rs) throws SQLException {
+		
+		Map<String, Bill> billMap = new LinkedHashMap<>();
```

Comment:
Is it possible to extract this data without having to import service related models?

Reply:
It has to be kept as a map of string and object which is going to slow down the application. if that's not a problem then it can be done.